src/media/css/footer.styl

```diff
@@ -47,17 +46,20 @@ $support-nav-height = 55px;
         }
     }
     .footer-content {
+        display: flex;
+        justify-content: space-between;
```

Comment:
I don't think `space-between` is going to do anything when one of the children has `flex-grow`.